src/test/java/com/checkmarx/flow/cucumber/integration/pullrequest/updatecomments/UpdatePullRequestCommentsSteps.java

```diff
@@ -58,24 +65,28 @@
     public static final String PULL_REQUEST_COMMENTS_URL = GITHUB_PR_BASE_URL + "/issues/"+ GITHUB_PR_ID + "/comments";
     private static final String GIT_URL = "https://github.com/cxflowtestuser/" + GIT_PROJECT_NAME;
     private static final String ADO_PR_COMMENTS_URL = "https://dev.azure.com/CxNamespace/d50fc6e5-a5ab-4123-9bc9-ccb756c0bf16/_apis/git/repositories/a89a9d2f-ab67-4bda-9c56-a571224c2c66/pullRequests/" + ADO_PR_ID + "/threads";
+    private final String filePath = "sample-sast-results" + "/" + "3-findings-filter-script-test.xml";
     private final GitHubService gitHubService;
     private final ADOService adoService;
     private GitHubController gitHubControllerSpy;
     private ADOController adoControllerSpy;
     private final ObjectMapper mapper = new ObjectMapper();
     private final GitHubProperties gitHubProperties;
     private final HelperService helperService;
-    private ScanResults scanResultsToInject;
+    private final ScaProperties scaProperties;
     private SourceControlType sourceControl;
     private FlowProperties flowProperties;
     private CxProperties cxProperties;
-    private String branchAdo;
+    private CxAuthClient authClient;
     private String branchGitHub;
     private ScannerType scannerType;
-    private final ScaProperties scaProperties;
 
+    private File sastFile;
+    private FilterConfiguration filterMedium = FilterConfiguration.fromSimpleFilters(Collections.singletonList(new Filter(Filter.Type.SEVERITY, "Medium")));
```

Comment:
why not refactor this function to get ellipsis of filters, and create the list inside, this will make it more readable, and less code.

Reply:
Fixed